{"timestamp":"2025-10-13T14:53:51.685343Z","level":"info","event":"DAG bundles loaded: dags-folder, example_dags","logger":"airflow.dag_processing.bundles.manager.DagBundlesManager","filename":"manager.py","lineno":179}
{"timestamp":"2025-10-13T14:53:51.685836Z","level":"info","event":"Filling up the DagBag from /opt/airflow/dags/ETL_toll_data.py","logger":"airflow.models.dagbag.DagBag","filename":"dagbag.py","lineno":593}
{"timestamp":"2025-10-13T14:53:51.714054Z","level":"info","event":"Task instance is in running state","logger":"task.stdout"}
{"timestamp":"2025-10-13T14:53:51.714197Z","level":"info","event":" Previous state of the Task instance: TaskInstanceState.QUEUED","logger":"task.stdout"}
{"timestamp":"2025-10-13T14:53:51.714343Z","level":"info","event":"Current task name:transform","logger":"task.stdout"}
{"timestamp":"2025-10-13T14:53:51.714412Z","level":"info","event":"Dag name:ETL_toll_data","logger":"task.stdout"}
{"timestamp":"2025-10-13T14:53:51.714664Z","level":"info","event":"Tmp dir root location: /tmp","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook","filename":"subprocess.py","lineno":78}
{"timestamp":"2025-10-13T14:53:51.715308Z","level":"info","event":"Running command: ['/usr/bin/bash', '-c', 'cd /opt/airflow/data/ouptut;tr [:lower:] [:upper:] < extracted_data.csv > tranformed_data.csv;']","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook","filename":"subprocess.py","lineno":88}
{"timestamp":"2025-10-13T14:53:51.728702Z","level":"info","event":"Output:","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook","filename":"subprocess.py","lineno":99}
{"timestamp":"2025-10-13T14:53:51.729996Z","level":"info","event":"/usr/bin/bash: line 1: cd: /opt/airflow/data/ouptut: No such file or directory","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook","filename":"subprocess.py","lineno":106}
{"timestamp":"2025-10-13T14:53:51.730271Z","level":"info","event":"/usr/bin/bash: line 1: extracted_data.csv: No such file or directory","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook","filename":"subprocess.py","lineno":106}
{"timestamp":"2025-10-13T14:53:51.730648Z","level":"info","event":"Command exited with return code 1","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook","filename":"subprocess.py","lineno":110}
{"timestamp":"2025-10-13T14:53:51.731000Z","level":"error","event":"Task failed with exception","logger":"task","filename":"task_runner.py","lineno":972,"error_detail":[{"exc_type":"AirflowException","exc_value":"Bash command failed. The command returned a non-zero exit code 1.","exc_notes":[],"syntax_error":null,"is_cause":false,"frames":[{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/task_runner.py","lineno":920,"name":"run"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/task_runner.py","lineno":1307,"name":"_execute_task"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/bases/operator.py","lineno":416,"name":"wrapper"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/operators/bash.py","lineno":232,"name":"execute"}],"is_group":false,"exceptions":[]}]}
{"timestamp":"2025-10-13T14:53:51.745417Z","level":"info","event":"Task instance in failure state","logger":"task.stdout"}
{"timestamp":"2025-10-13T14:53:51.745756Z","level":"info","event":"Task start","logger":"task.stdout"}
{"timestamp":"2025-10-13T14:53:51.745872Z","level":"info","event":"Task:<Task(BashOperator): transform>","logger":"task.stdout"}
{"timestamp":"2025-10-13T14:53:51.745931Z","level":"info","event":"Failure caused by Bash command failed. The command returned a non-zero exit code 1.","logger":"task.stdout"}
{"timestamp":"2025-10-13T14:53:51.794328Z","level":"warning","event":"Using Connection.get_connection_from_secrets from `airflow.models` is deprecated.Please use `get` on Connection from sdk(`airflow.sdk.Connection`) instead","category":"DeprecationWarning","filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/models/connection.py","lineno":486,"logger":"py.warnings"}
{"timestamp":"2025-10-13T14:53:51.800077Z","level":"info","event":"Email alerting: attempt 1","logger":"airflow.utils.email","filename":"email.py","lineno":261}
{"timestamp":"2025-10-13T14:53:51.807034Z","level":"info","event":"Email alerting: attempt 1","logger":"airflow.utils.email","filename":"email.py","lineno":261}
{"timestamp":"2025-10-13T14:53:51.807666Z","level":"error","event":"Top level error","logger":"task","filename":"task_runner.py","lineno":1457,"error_detail":[{"exc_type":"ConnectionRefusedError","exc_value":"[Errno 111] Connection refused","exc_notes":[],"syntax_error":null,"is_cause":false,"frames":[{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/task_runner.py","lineno":1452,"name":"main"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/task_runner.py","lineno":1419,"name":"finalize"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/task_runner.py","lineno":1259,"name":"_send_task_error_email"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/email.py","lineno":79,"name":"send_email"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/email.py","lineno":154,"name":"send_email_smtp"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/email.py","lineno":263,"name":"send_mime_email"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/email.py","lineno":306,"name":"_get_smtp_connection"},{"filename":"/usr/python/lib/python3.12/smtplib.py","lineno":255,"name":"__init__"},{"filename":"/usr/python/lib/python3.12/smtplib.py","lineno":341,"name":"connect"},{"filename":"/usr/python/lib/python3.12/smtplib.py","lineno":312,"name":"_get_socket"},{"filename":"/usr/python/lib/python3.12/socket.py","lineno":865,"name":"create_connection"},{"filename":"/usr/python/lib/python3.12/socket.py","lineno":850,"name":"create_connection"}],"is_group":false,"exceptions":[]},{"exc_type":"ConnectionRefusedError","exc_value":"[Errno 111] Connection refused","exc_notes":[],"syntax_error":null,"is_cause":false,"frames":[{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/task_runner.py","lineno":1257,"name":"_send_task_error_email"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/email.py","lineno":79,"name":"send_email"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/email.py","lineno":154,"name":"send_email_smtp"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/email.py","lineno":263,"name":"send_mime_email"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/email.py","lineno":306,"name":"_get_smtp_connection"},{"filename":"/usr/python/lib/python3.12/smtplib.py","lineno":255,"name":"__init__"},{"filename":"/usr/python/lib/python3.12/smtplib.py","lineno":341,"name":"connect"},{"filename":"/usr/python/lib/python3.12/smtplib.py","lineno":312,"name":"_get_socket"},{"filename":"/usr/python/lib/python3.12/socket.py","lineno":865,"name":"create_connection"},{"filename":"/usr/python/lib/python3.12/socket.py","lineno":850,"name":"create_connection"}],"is_group":false,"exceptions":[]}]}
{"timestamp":"2025-10-13T14:53:51.816826Z","level":"warning","event":"Process exited abnormally","exit_code":1,"logger":"task"}
